tensorflow/core/api_def/base_api/api_def_ArgMin.pbtxt

```diff
@@ -11,5 +11,8 @@ END
   summary: "Returns the index with the smallest value across dimensions of a tensor."
   description: <<END
 Note that in case of ties the identity of the return value is not guaranteed.
+
+For example:
+
```

Comment:
`ArgMax.pbtxt` has a full example listed under `Usage`, this one doesn't and also seems incomplete.